Light up bot/lightupbot.py

```diff
@@ -0,0 +1,92 @@
+import asyncio
+import os
+
+from viam.robot.client import RobotClient
+from viam.rpc.dial import Credentials, DialOptions
+from viam.components.camera import Camera
+from viam.components.types import CameraMimeType
+from viam.services.vision import VisionServiceClient, VisModelConfig, VisModelType, Detection
+from PIL import ImageDraw
+from kasa import Discover, SmartPlug
+
+#these must be set, you can get them from your robot's 'CODE SAMPLE' tab
+robot_secret = os.getenv('ROBOT_SECRET') or ''
+robot_address = os.getenv('ROBOT_ADDRESS') or ''
+
+async def connect():
+    creds = Credentials(
+        type='robot-location-secret',
+        payload=robot_secret)
+    opts = RobotClient.Options(
+        refresh_interval=0,
+        dial_options=DialOptions(credentials=creds)
+    )
+    return await RobotClient.at_address(robot_address, opts)
+
+#for displaying your detections with a red box around them, if needed
+def drawBox(d, frame):
+  rect = [d.x_min, d.y_min, d.x_max, d.y_max]
+  img1 = ImageDraw.Draw(frame)
+  img1.rectangle(rect, outline="red")
+  return frame
+
+async def main():
+    robot = await connect()
+
+    print('Resources:')
+    print(robot.resource_names)
+
+    camera = Camera.from_robot(robot, "camera")
+    image = await camera.get_image()
+
+    #get and setup vision service
+    #replace the "model_path" to where your tflite package lives, and "label_path" to where your text file lives on the robot. 
+    vision = VisionServiceClient.from_robot(robot)
+    params = {"model_path": "/location/of/tflite/effdet0.tflite", "label_path": "/location/of/labels/labels.txt", "num_threads": 1}
```

Comment:
A relative path like "./effdet0.tflite" should work since the script and model and labels live in the same location.  This way people can use this script without having to modify lines of code.